### DISCUSSION TITLE
Permissions denied via sftp

### DISCUSSION THREAD

**Discussion by DiaxuDev** [itzg/docker-minecraft-server#1803] [Q&A] [answered]
I have a problem. When I connect to the data directory via SFTP, then enter plugins, I can modify files. But when a new directory is created by a new plugin, I can't modify files in there. I tried using `chmod` and `chgrp` with recursive flag, but it worked only for directories created before this command. I'm new to docker, and I'm wondering if there's a way to automatically set permissions?

**Comment by itzg** [comment, marked as answer]
The container runs the server with  user ID 1000 by default. Make sure you're working on the files as a user with the same UID. More info at https://github.com/itzg/docker-minecraft-server#running-as-alternate-usergroup-id
